dev-packages/node-integration-tests/suites/express/sentry-trace/baggage-header-assign/test.ts

```diff
@@ -105,9 +103,7 @@ test.skip('Should populate and propagate sentry baggage if sentry-trace header d
   });
 });
 
-// TODO(v8): Fix this test
-// eslint-disable-next-line jest/no-disabled-tests
-test.skip('Should populate Sentry and ignore 3rd party content if sentry-trace header does not exist', async () => {
+test('Should populate Sentry and ignore 3rd party content if sentry-trace header does not exist', async () => {
```

Comment:
maybe I'm missunderstanding the test but looking at the assertions below, shouldn't we also check for the presence of `foo=bar,bar=baz`?

Reply:
Oh I think this is fine; We should only care about not overwriting baggage entries that were already added to an outgoing request before we inject our entries. The `foo` and `bar` entries were not added further; when the outgoing request was made, right?